python/ray/rllib/train.py

```diff
@@ -126,10 +126,8 @@ def run(args, parser):
         cluster = Cluster()
         for _ in range(args.ray_num_nodes):
             cluster.add_node(
-                resources={
-                    "num_cpus": args.ray_num_cpus or 1,
-                    "num_gpus": args.ray_num_gpus or 0,
-                },
+                num_cpus=args.ray_num_cpus or 1,
```

Comment:
@ericl this change is unrelated to this PR, but this seemed like a potential bug.

Reply:
Seems fine -- @richardliaw what was the motivation for resources in the first place?

Reply:
`resources` should be for custom resources, it was just accidentally used incorrectly here. Also, from ray_params.py, this error message is pretty misleading:

```
    def _check_usage(self):
        if self.resources is not None:
            assert "CPU" not in self.resources, (
                "'CPU' should not be included in the resource dictionary. Use "
                "num_cpus instead.")
            assert "GPU" not in self.resources, (
                "'GPU' should not be included in the resource dictionary. Use "
                "num_gpus instead.")
```

Reply:
@robertnishihara this fix is good though